.golangci.yml

```diff
@@ -6,26 +6,34 @@ run:
   - "zz_generated.*\\.go$"
   - ".*conversion.*\\.go$"
 linters:
+  disable-all: true
   enable:
+    - deadcode
+    - goconst
+    - gofmt
     - golint
+    - gosec
     - govet
-    - gofmt
-    - structcheck
-    - varcheck
-    - interfacer
-    - unconvert
     - ineffassign
-    - goconst
-    # - gocyclo
+    - interfacer
     - maligned
     - misspell
     - nakedret
     - prealloc
-    - gosec
-    - deadcode
-  disable-all: true
+    - structcheck
+    - unconvert
+    - varcheck
   # Run with --fast=false for more extensive checks
   fast: true
-issue:
+issues:
+  exclude-rules:
+    - path: internal/test/
+      linters:
+        - golint
+    - path: cloudtest\.go
+      linters:
+        - golint
+  include:
+  - EXC0002 # include "missing comments" issues from golint
+  max-issues-per-linter: 0
   max-same-issues: 0
-  max-per-linter: 0
```

Comment:
And this should have been [`max-issues-per-linter`](https://golangci-lint.run/usage/configuration/#config-file)--fixed now.